README.md

```diff
@@ -32,6 +32,8 @@ The backend of the project is built on spring-boot and there is a dockerfile pre
 
 #### Steps to spin up the backend
 
+## Using Dockerfile
+
 a) Create a package: 
 ```
 mvn clean package
```

Comment:
I think that this is going to have be done in docker most likely using the "builder" pattern so future users won't need to install mvn and other tools.  Arguably since they're working with Jenkins, they have Java already installed, but we can't assume they have maven on their machine.